flink-connectors/flink-hadoop-compatibility/src/main/scala/org/apache/flink/api/scala/hadoop/mapred/HadoopInputFormat.scala

```diff
@@ -22,6 +22,17 @@ import org.apache.flink.api.java.hadoop.mapred.HadoopInputFormatBase
 
 import org.apache.hadoop.mapred.{InputFormat, JobConf}
 
+/**
+ * @deprecated
+ *   All Flink Scala APIs are deprecated and will be removed in a future Flink version version. You
```

Comment:
```suggestion
 *   All Flink Scala APIs are deprecated and will be removed in a future Flink major version. You
```

Reply:
Oh my -- I'll fix all of these, of course.  Good catch on this!